apps/console/webpack.config.ts

```diff
@@ -335,9 +403,10 @@ module.exports = (config: WebpackOptionsNormalized, context: NxWebpackContextInt
     }
 
     // Update the existing `CopyPlugin` plugin added by NX.
-    const existingCopyPlugin = config.plugins.find((plugin) => {
-        return plugin.constructor.name === "CopyPlugin";
-    });
+    const existingCopyPlugin: webpack.WebpackPluginInstance = 
```

Comment:
`WebpackPluginInstance` should already be there.

```suggestion
    const existingCopyPlugin: WebpackPluginInstance = 
```